### ISSUE TITLE
[CAL-273] BUG: Reordering event types not working

### ISSUE THREAD

**Issue by Udit-takkar** [calcom/cal.com#5255]
[bfd896dd-38ba-4042-8780-64fe2c058eda](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/1d541d8f-b66e-42eb-8a33-337db5a135b8/bfd896dd-38ba-4042-8780-64fe2c058eda)

> From [Linear-GitHub Sync](https://synclinear.com)

**Comment by Toledodev** [comment]
Hey @Udit-takkar please check [this video](https://www.loom.com/share/07a74f8cdc604184aa90166a549a2ddf), I think this bug is not happening anymore

**Comment by Udit-takkar** [comment]
ok let me try again on prod

**Comment by Udit-takkar** [comment]
https://user-images.githubusercontent.com/53316345/198852742-40f1a2d4-1751-4050-ac60-8d82e1f848af.mov

idk still not working on latest main for me

**Comment by emrysal** [comment]
Confirmed here as well @Udit-takkar - this is an issue on main and has not made it into SaaS production yet.
